### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by KingLyrics**
Just added the footer component. Corrections highly needed.

### CODE REVIEW

File: src/sass/main.scss [hunk 1]
```diff
@@ -206,3 +206,38 @@ body {
...
 	}
 }
+
+
+// footer
+
+.foot-container{
+	margin-top: 4rem;
+
+	& .logo{
```
Comment (gmagnenat) [inline src/sass/main.scss]: don't need the & here as you are just nesting.
& is needed if you want to add something at the end of the parent tag for example.

File: src/sass/main.scss [hunk 2]
```diff
@@ -206,3 +206,38 @@ body {
...
+	}
+
+	& .center-text{
+		width: 295px;
+		margin-top: 1rem;
+		p{
+			line-height: $lh-14;
+		}
+		span{
+			font-weight: bold;
```
Comment (gmagnenat) [inline src/sass/main.scss]: use the scss variables $fw-bold and $fw-regular

File: src/sass/main.scss [hunk 3]
```diff
@@ -206,3 +206,38 @@ body {
...
+		}
+	}
+
+	.socials{
+		padding-top: 3rem;
+		padding-bottom: 2rem;
+		display: flex;
+
+		& a{
```
Comment (gmagnenat) [inline src/sass/main.scss]: don't need the &

File: src/sass/main.scss [hunk 4]
```diff
@@ -206,3 +206,38 @@ body {
...
+// footer
+
+.foot-container{
+	margin-top: 4rem;
+
+	& .logo{
+		padding-bottom: $fs-12;
+	}
+
+	& .center-text{
```
Comment (gmagnenat) [inline src/sass/main.scss]: don't need the &

File: src/sass/main.scss (hunk 1 above)
Comment (KingLyrics) [reply src/sass/main.scss]: alright noted

File: src/sass/main.scss (hunk 2 above)
Comment (KingLyrics) [reply src/sass/main.scss]: Corrected

File: src/sass/main.scss (hunk 3 above)
Comment (KingLyrics) [reply src/sass/main.scss]: Resolved

File: src/sass/main.scss (hunk 4 above)
Comment (KingLyrics) [reply src/sass/main.scss]: Corrected

**Comment by KingLyrics** [comment]
All changes have been made.

**Review by gmagnenat — APPROVED** [review]

**Pull request merged**